src/OpenSage.Game/Graphics/Cameras/RtsCameraController.cs

```diff
@@ -204,6 +210,23 @@ void ICameraController.UpdateCamera(Camera camera, in CameraInputState inputStat
                 newPosition,
                 targetPosition,
                 Vector3.UnitZ);
+
+            CameraLookData lookData = createLookData(newPosition, targetPosition);
+            checkForCameraPositionSave(inputState, lookData);
+            checkForCameraPositionLoad(inputState, lookData, camera);
+        }
+
+        private CameraLookData createLookData(Vector3 newPosition, Vector3 targetPosition)
+        {
+            CameraLookData lookData = new CameraLookData();
```

Comment:
I'd suggest an object initializer: https://docs.microsoft.com/de-de/dotnet/csharp/programming-guide/classes-and-structs/how-to-initialize-objects-by-using-an-object-initializer

Also use var here again